### ISSUE TITLE
There was an error with the server. Sorry about this! It'll probably be fixed by the next day.

### ISSUE THREAD

**Issue by Ads20000** [cnguy/onetricks.net#18]
Think this has been happening for quite a while now, the pages are very out-of-date (possibly as out-of-date as last season). See Tryndamere for example, I don't think Fogged has climbed much with twtvfogged this season yet that account is listed as Masters?

**Comment by cnguy** [comment]
yep, basically i haven't been working on this (in terms of production) for the past season since i've been really busy (school sucks). i'm mostly just refactoring the internal stuffs to make it work better. ETA probably 2 weeks to get something good showing again :)

**Comment by cnguy** [comment]
A lot of stats should be updated for the Challenger and Grandmaster leagues! Master is a work-in-progress. Server is still down as I have to update a lot of stats, but I can now see `riste`!

**Comment by Ads20000** [comment]
Update: the site is currently completely empty in terms of the po
![Screenshot from 2019-03-30 02-04-47](https://user-images.githubusercontent.com/3178485/55269884-36393d80-5290-11e9-8826-2739e90ec5b1.png)
pulation of one-tricks on the page
